components/nodemanager-service/pgdb/project_update.go

```diff
@@ -123,7 +117,20 @@ func (db *DB) UpdateProjectTags(ctx context.Context,
 
 // JobStatus - get the job status of the project update
 func (db *DB) JobStatus(ctx context.Context, jobID string) (project_update_lib.JobStatus, error) {
```

Comment:
This is where we can check the status of both a project update and project delete status.